docs/application/tizen-studio/release-notes/4-1-release-notes.md

```diff
@@ -0,0 +1,61 @@
+# Tizen Studio 4.1 Release Notes
+
+- Release Date: Nov 23, 2020
+
+## IDE and Tools
+
+### New Features
+
+- IDE
+  - Tizen 6.0 platform support has been added.
+  - Battery Historian tool has been added to help analyze the battery usage.
+  - Tizen Core new CLI support for Tizen 6.0 Native applications has been added.
+  - Toolchain has been updated to GCC 9.2 and LLVM 10.
+  - Support for macOS Catalina, Ubuntu 18.04, and Ubuntu 20.04 has been added.
+  - Support for AArch64 architecture on IoT 6.0 extension for RPi4 has been added.
+
+### Deprecated
+   - Tizen RT has been deprecated from Tizen Studio 4.1.
+   - Bridge Networking support in emulators will be deprecated in future Tizen Studio releases.
+  
+### Known Issues
+
+- Tizen Studio
+  - In macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.
+  - In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+  - Native Templates-5.5 will not build with CLI when compiler is set to **gcc**.
```

Comment:
@manishtr This issue is only with macos Catalina and above. Can we capture the same here? This issue will not be present in Windows or MacOS.
We can also provide Guide to fix the same?

Reply:
applied, we will create a page in https://developer.tizen.org/community/tip-tech for it, thanks.